src/components/Book/styles.module.scss

```diff
@@ -0,0 +1,22 @@
+@import 'variables/colors';
+
+.book {
+  border: 1px solid $gray-main;
+  box-shadow: 0 0 13px 0 rgba(50, 50, 50, 0.91);
```

Comment:
We don't use RGB, we always use hex. You can convert this to a hex, define the constant in colors and use it here like this: `rgba($someColor, 0.91)`